src/content.rs

```diff
@@ -105,7 +105,7 @@ impl Comment {
                     let is_last = index == len - 1;
                     if is_first {
                         formatter.append(&format!("/*{}", line))?;
-                    } else if line.len() > 0 {
```

Comment:
I don't think you meant to make this change, right?

This came from a recently-merged PR, which your PR may have wiped out by mistake. Try rebasing from upstream.

Reply:
Ack, good catch! Yeah, this wasn't intentional and is mostly a consequence of bad version control practices. I was making changes in the json5format code in third_party/ to see what I needed to get my code to build... and then I copied that code into my local copy of the github version. Other stuff is the same deal. I'll get those changes out of this PR.